apps/api/src/resources/Sig/entity.ts

```diff
@@ -19,6 +19,10 @@ export class Sig extends BaseEntity {
   @Field()
   @PrimaryColumn()
   public name: string;
+  
+  @Field({nullable: true})
+  @Column({nullable: true})
+  public topic: string; //shorthand version of the original name
```

Comment:
topic doesn't seem like a good name if its supposed to be the shortened name. Also I noticed that none of the inserted values in the migration have topics